src/Bathymetry.jl

```diff
@@ -36,6 +36,28 @@ end
 etopo_url = "https://www.dropbox.com/scl/fi/6pwalcuuzgtpanysn4h6f/" *
             "ETOPO_2022_v1_60s_N90W180_surface.nc?rlkey=2t7890ruyk4nd5t5eov5768lt&st=yfxsy1lu&dl=0"
 
+"""
+    download_bathymetry(; dir = download_bathymetry_cache,
+                          url = etopo_url,
+                          filename = "ETOPO_2022_v1_60s_N90W180_surface.nc")
+
+Download the bathymetry from `url` and saves it under `filename` in the directory `dir` and
+return the full filepath where the bathymetry is saved.
+"""
+function download_bathymetry(;
+                             url = etopo_url,
+                             dir = download_bathymetry_cache,
+                             filename = "ETOPO_2022_v1_60s_N90W180_surface.nc")
+
+    filepath = joinpath(dir, filename)
+    # No need for @root here, because only rank 0 accesses this function
```

Comment:
Is this comment pertinent?

```suggestion

```

Reply:
Uh. I think it's wrong. We need @root. This is just an assumption that isn't always true and I don't know why one would make this assumption or how it is enforced.

Reply:
Hm... Should we rephrase the comment? or leave it there for now..?

Reply:
No we should add @ root.